tokio/src/sync/mpsc/bounded.rs

```diff
@@ -1060,6 +1059,46 @@ impl<T> Sender<T> {
         self.chan.is_closed()
     }
 
+    /// Checks if a channel is empty.
+    ///
+    /// This method returns `true` if the channel has no messages.
+    ///
+    /// # Examples
+    /// ```
+    /// use tokio::sync::mpsc;
+    ///
+    /// #[tokio::main]
+    /// async fn main() {
+    ///     let (tx, rx) = mpsc::channel(10);
+    ///     assert!(rx.is_empty());
+    ///
+    ///     tx.send(0).await.unwrap();
+    ///     assert!(!rx.is_empty());
```

Comment:
This is a method on the sender, so the example should call it on the sender, not the receiver.